cupy/statistics/order.py

```diff
@@ -52,10 +58,56 @@ def amax(a, axis=None, out=None, keepdims=False, dtype=None):
     return a.max(axis=axis, dtype=dtype, out=out, keepdims=keepdims)
 
 
-# TODO(okuta): Implement nanmin
+def nanmin(a, axis=None, out=None, keepdims=False):
+    """Return the minimum of an array along an axis ignoring NaN.
 
+    When all NaN slices are found, a :class:`RuntimeWarning` is raised and NaN
```

Comment:
Does "all Nan slices" mean a slice whose elements are all NaN? If so, rewrite it as `When there is a slice whose elements are all NaN, ...`.